pkg/agent/cniserver/pod_configuration.go

```diff
@@ -325,6 +326,17 @@ func (pc *podConfigurator) removeInterfaces(containerID string) error {
 		return err
 	}
 
+	// Since only VLAN and SR-IOV interfaces are supported by now, we judge the
```

Comment:
VLAN -> OVS ports

Remove interfaces can be called for primary interfaces too, which are not VLAN interfaces.